packages/pwa/app/translations/en-GB.json

```diff
@@ -1,1410 +1,1410 @@
-  },
-  "use_registration_fields.error.required_first_name": {
-    "defaultMessage": "Please enter your first name."
-  },
-  "use_registration_fields.error.required_last_name": {
-    "defaultMessage": "Please enter your last name."
-  },
-  "use_registration_fields.error.required_password": {
-    "defaultMessage": "Please create a password."
-  },
-  "use_registration_fields.error.special_character": {
-    "defaultMessage": "Password must contain at least one special character."
-  },
-  "use_registration_fields.error.uppercase_letter": {
-    "defaultMessage": "Password must contain at least one uppercase letter."
-  },
-  "use_registration_fields.label.email": {
-    "defaultMessage": "Email"
-  },
-  "use_registration_fields.label.first_name": {
-    "defaultMessage": "First Name"
-  },
-  "use_registration_fields.label.last_name": {
-    "defaultMessage": "Last Name"
-  },
-  "use_registration_fields.label.password": {
-    "defaultMessage": "Password"
-  },
-  "use_registration_fields.label.sign_up_to_emails": {
-    "defaultMessage": "Sign me up for Salesforce emails (you can unsubscribe at any time)"
-  },
-  "use_reset_password_fields.error.required_email": {
-    "defaultMessage": "Please enter a valid email address."
-  },
-  "use_reset_password_fields.label.email": {
-    "defaultMessage": "Email"
-  },
-  "use_update_password_fields.error.contain_number": {
-    "defaultMessage": "Password must contain at least one number."
-  },
-  "use_update_password_fields.error.lowercase_letter": {
-    "defaultMessage": "Password must contain at least one lowercase letter."
-  },
-  "use_update_password_fields.error.minimum_characters": {
-    "defaultMessage": "Password must contain at least 8 characters."
-  },
-  "use_update_password_fields.error.required_new_password": {
-    "defaultMessage": "Please provide a new password."
-  },
-  "use_update_password_fields.error.required_password": {
-    "defaultMessage": "Please enter your password."
-  },
-  "use_update_password_fields.error.special_character": {
-    "defaultMessage": "Password must contain at least one special character."
-  },
-  "use_update_password_fields.error.uppercase_letter": {
-    "defaultMessage": "Password must contain at least one uppercase letter."
-  },
-  "use_update_password_fields.label.current_password": {
-    "defaultMessage": "Current Password"
-  },
-  "use_update_password_fields.label.new_password": {
-    "defaultMessage": "New Password"
-  },
-  "wishlist_primary_action.info.added_to_cart": {
-    "defaultMessage": "{quantity} {quantity, plural, one {item} other {items}} added to cart"
-  },
-  "wishlist_secondary_button_group.action.remove": {
-    "defaultMessage": "Remove"
-  },
-  "wishlist_secondary_button_group.info.item_removed": {
-    "defaultMessage": "Item removed from wishlist"
-  },
-  "with_registration.info.please_sign_in": {
-    "defaultMessage": "Please sign in to continue!"
-  }
+    "account.accordion.button.my_account": {
```

Comment:
Please check your prettier config, looks like there is no change on this file except indentation.

Reply:
Reverted formatting changes